### PR TITLE
Add `multiEntry` support to extract runnable items

### PR DESCRIPTION
**Pull request by codenirvana**
**Behavior:**
- Follows the same order as of provided endpoints
- Extract nested entry points also.

**Todo:**
- [ ] Check for edge cases
- [ ] Add Unit Tests
- [ ] Integrate with `newman`

### CODE REVIEW

**Comment by kunagpal** [comment]
@codenirvana Windows tests failing.

File: lib/runner/extract-runnable-items.js [hunk 1]
```diff
@@ -61,6 +61,56 @@ var sdk = require('postman-collection'),
...
+            entry = item.name;
+            index = state.pendingEntries.indexOf(entry);
+
+            // search by id if item name not found.
+            if (index === -1) {
+                entry = item.id;
+                index = state.pendingEntries.indexOf(entry);
+            }
+
+            if (index !== -1) {
```
[the remark this answers is not in the document]
Comment (codenirvana) [reply lib/runner/extract-runnable-items.js]: `index` might be updated in the above `if`, so need to test again.

File: lib/runner/extract-runnable-items.js [hunk 2]
```diff
@@ -61,6 +61,56 @@ var sdk = require('postman-collection'),
...
+     * @param {Array<Item|ItemGroup>} state.items
+     * @param {Object} state.reference
+     * @returns {Object}
+     */
+    findItemsFromEntries = function (itemGroup, state) {
+        var key,
+            index,
+            entry;
+
+        itemGroup.items.each(function (item) {
```
[the remark this answers is not in the document]
Comment (codenirvana) [reply lib/runner/extract-runnable-items.js]: Will check for the edge cases in the first call. For recursive calls, it pre-checks for this `L108`.

**Comment by codenirvana** [comment]
#619
